package.json

```diff
@@ -35,6 +35,7 @@
     "wds": "wds --app-index dev/index.html --watch --node-resolve",
     "preserve": "yarn prep",
     "serve": "concurrently -c \"cyan,blue\" \"yarn:rollup.watch\" \"yarn:wds\" --kill-others",
+    "storybook.axe": "yarn storybook.prod && yarn axe-storybook --build-dir=storybook-static/",
```

Comment:
What about `test.accessibility` as the name. The rest of the storybook commands are about running storybook. This command, to me falls under the testing category.

Reply:
Interesting. This accessibility test runs against storybook stories.

Reply:
@damontgomery you may be correct. And we may want a better naming scheme for some of these things (all of them) as we move forward. 

For instance in our `fin` scripts, we stopped using the names of a tool specifically (like Storybook), and abstracted the command names to be about the result. For now we'll go with this, but I know we have a ticket somewhere to "review" the `package.json` for ease of use and all that good stuff.